packages/server/src/tests/epics/topReaderArticleCountBadge/build.ts

```diff
@@ -0,0 +1,71 @@
+import { epic, epicTopReaderAcBadgeV1, epicTopReaderAcBadgeV2 } from '@sdc/shared/dist/config';
+import { CountryGroupId } from '@sdc/shared/dist/lib';
+import { EpicTest, SecondaryCtaType } from '@sdc/shared/dist/types';
+import { Copy, CTA } from './copy';
+
+interface Options {
+    suffix: string;
+    locations: CountryGroupId[];
+    copy: Copy;
+}
+
+export function buildTest({ suffix, locations, copy }: Options): EpicTest {
+    return {
+        name: `TopReaderArticleCountBadge__${suffix}`,
+        campaignId: `TopReaderArticleCountBadge__${suffix}`,
+        hasArticleCountInCopy: true,
+        isOn: false,
```

Comment:
We can create a follow up pr to change this to `true` and merge that when we want to go live